jenkins/opensearch_dashboards/Jenkinsfile

```diff
@@ -0,0 +1,143 @@
+pipeline {
+    agent none
+    environment {
+        OPENSEARCH_BUILD_ID = "${BUILD_NUMBER}"
+    }
+    triggers {
+      parameterizedCron '''
+        H */2 * * * %INPUT_MANIFEST=1.1.0/opensearch-dashboards-1.1.0.yml
```

Comment:
This will trigger every 2 hours, is that enough/not enough for your use case, how long to the builds take?